examples/polkadot.ts

```diff
@@ -6,14 +7,14 @@ import { Keyring } from '@polkadot/api';
 import { cryptoWaitReady } from '@polkadot/util-crypto';
 
 import {
-  createSignedTx,
```

Comment:
Can we revert these types of changes?

Reply:
Yes, we can go back to two spaces - but I think switching to tabs of 2 space size is more consistent with other parity js work.

Reply:
In general I favor consistency, but in general spaces vs tabs is a holy war and I'm decidedly on the spaces and in particular belong to the 2-space sect LOL Furthermore, I think in this case we can argue that we are sticking with the consistency _of this repo_ :relieved: